src/components/Personalization/turbine/executeRules.js

```diff
@@ -10,26 +10,28 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
```

Comment:
A little unit test for this module would be great.

Reply:
Done.